netmiko/huawei/huawei_smartax.py

```diff
@@ -164,3 +207,15 @@ def cleanup(self, command: str = "quit") -> None:
                 return
             time.sleep(0.01)
         raise ValueError("Failed to log out of the device")
+
+
+class HuaweiSmartAXSSHMMI(HuaweiSmartAXSSH):
+
+    def session_preparation(self) -> None:
+        super().session_preparation()
+        self._enter_mmi_mode()
+
+    def cleanup(self, command: str = "quit") -> None:
+        if self.check_config_mode():
+            self._send_command_str("quit")
+        self._send_command_str("quit")
```

Comment:
Don't you need to call the parent class for the `.cleanup()` call? There are a bunch of things that are missing here--if we don't call the parent classes (for .cleanup()).